lib/generateMarkdown.js

```diff
@@ -42,11 +42,14 @@ function generateMarkdown(options) {
 
     schema = resolved.schema;
     var orderedTypes = sortObject(resolved.referencedSchemas);
+    for (var type in orderedTypes) {
+        orderedTypes[type].children.sort();
+    }
 
     // We need the reverse-sorted version so that when we do type searching we find the longest type first.
     var orderedTypesDescending = sortObject(resolved.referencedSchemas, false);
 
-    md += getTableOfContentsMarkdown(schema, orderedTypes, options.headerLevel);
+     md += getTableOfContentsMarkdown(schema, orderedTypes, options.headerLevel);
```

Comment:
I've gone through and used Shift-Alt-F to let VS Code auto format the documents.